emails/models.py

```diff
@@ -297,6 +297,16 @@ def has_phone(self):
                 return True
         return False
 
+    @property
+    def has_vpn(self):
+        if not self.fxa:
+            return False
+        user_subscriptions = self.fxa.extra_data.get("subscriptions", [])
+        for sub in settings.SUBSCRIPTIONS_WITH_VPN.split(","):
+            if sub in user_subscriptions:
+                return True
+        return False
```

Comment:
optimization (non-blocking): We should refactor all 3 of these into a `has_subscription(self)` function (non-property?) that memo-izes the results in some way so if calling code needs to check multiple subscription values, we dont' need to make multiple `self.fxa` (i.e., DB) calls.

Reply:
Hmm OK, so I started trying to implement this, but then I chickened out because I was afraid of how this would interact with `profile_refresh` - not just the FxA data in the database could get stale (requiring that call), but then the in-memory cache could get stale as well. I'm guessing we could expose a method on the `Profile` model to clear the memoised values, which could be called by `profile_refresh`, but I'd like a second opinion on that approach first :)

Reply:
No worries on it - we can make it a follow-up.